substrate/primitives/src/sandbox.rs

```diff
@@ -180,6 +180,23 @@ impl Slicable for ReturnValue {
 	}
 }
 
+impl ReturnValue {
+	/// Maximum number of bytes `ReturnValue` might occupy when serialized with
+	/// `Slicable`.
+	///
+	/// Breakdown:
+	///  1 byte for encoding unit/value variant
+	///  1 byte for encoding value type
+	///  8 bytes for encoding the biggest value types: f64, i64.
+	pub const ENCODED_MAX_SIZE: usize = 10;
```

Comment:
I'm not entirely sure about this. On the one hand, this doesn't look too reliable. On the other hand, dynamic approach (i.e when runtime allocates a memory for the return status and value, and returns it) seems more complex and less performant.

Reply:
`i128` is now the biggest (primitive) value type...

Reply:
Clarified the comment, I hope it's better now!